migrations/v6-MIGRATION.md

```diff
@@ -30,20 +40,19 @@
 
 ### StoreProduct changes
 
-`StoreProduct` has been made an interface, which `GoogleStoreProduct` and `AmazonStoreProduct` implement. 
-`StoreProduct` is no longer `Parcelable`.
```

Comment:
Hmm I think it's worth mentioning this. Some devs implementations might rely on this.